tests/test_eggshell.py

```diff
@@ -0,0 +1,259 @@
+import pytest
+
+from .common import TESTDATA
+
+from os.path import basename, join
+import tempfile
+import tarfile
+import zipfile
+from netCDF4 import Dataset
+
+from eggshell import utils
+from eggshell.utils import local_path
+from eggshell.nc import ocg_utils
+from eggshell.nc import nc_utils
+from eggshell.config import Paths
+import flyingpigeon as fp
+paths = Paths(fp)
+
+
+def test_Paths():
+    assert "flyingpigeon/tests/testdata" in paths.testdata
+    assert 'flyingpigeon/data' in paths.data
+    assert 'flyingpigeon/data/shapefiles' in paths.shapefiles
+
+
+def test_local_path():
+    assert local_path('file:///tmp/test.nc') == '/tmp/test.nc'
+    assert local_path('/tmp/test.nc') == '/tmp/test.nc'
+
+
+def test_ocgis_import():
+    from ocgis import constants
+
+
+# def test_has_Lambert_Conformal():
+#     has_lambert = ocg_utils.has_Lambert_Conformal(
+#         [local_path(TESTDATA['cordex_tasmax_2006_nc']),
+#          local_path(TESTDATA['cordex_tasmax_2007_nc'])])
+#     assert False == has_lambert
+
+
+def test_gdal():
+    from flyingpigeon.subset import clipping
+
+
+@pytest.mark.skip(reason="no way of currently testing this")
+def test_download_with_cache():
+    filename = utils.download(TESTDATA['cmip5_tasmax_2006_nc'], cache=paths.cache)
+    assert basename(filename) == 'tasmax_Amon_MPI-ESM-MR_rcp45_r1i1p1_200601-200612.nc'
+
+
+def test_archive_tar():
+    result = utils.archive(local_path(TESTDATA['cmip5_tasmax_2006_nc']),
+                           format='tar',
+                           output_dir=tempfile.mkdtemp())
+    tar = tarfile.open(result)
+    assert len(tar.getnames()) == 1
+
+#  [local_path(TESTDATA['cmip5_tasmax_2007_nc'])],
+
+
+def test_archive_zip():
+    result = utils.archive(local_path(TESTDATA['cmip5_tasmax_2006_nc']),
+                           format='zip',
+                           output_dir=tempfile.mkdtemp())
+    zipf = zipfile.ZipFile(result)
+    assert len(zipf.namelist()) == 1
+
+
+def test_get_frequency():
+    freq = nc_utils.get_frequency(local_path(TESTDATA['cmip5_tasmax_2007_nc']))
```

Comment:
move eggshell tests to eggshell.

Reply:
DONE, moved to eggshell.

Eggshell test in eggshell should than be organized in the same structure like eggshell itself, right? 
like 
`test_eggshell_nc_utils`
`test_eggshell_ogc_utils`
`test_eggshell_plot`
etc... 

How to call the tests? In eggshell or out of flyingpigeon or both??

Reply:
`eggshell` is a library on its own ... not knowing about fp. Just call `pytest` ... see eggshell dev guide.